backend/schema/corpora_schema_h5ad_implementation.md

```diff
@@ -0,0 +1,78 @@
+# Corpora Schema AnnData Implementation
+
+Corpora datasets are stored in the HDF5-backed AnnData format as written by version 0.7 of the anndata library. One
+goal of using this format is to have all metadata stored in the same file as the data, so cellxgene will have access to
+all Corpora metadata within a single file.
+
+The Corpora schema requirements and definitions for the AnnData `X`, `uns`, `obs`, and `obsm` attributes are below.
+
+
+### Note on types
+The types below are python3 types. Note that a python3 `str` is a sequence of Unicode code points, which is stored
+null-terminated and UTF-8-encoded by anndata.
+
+For succinctness, there is a `link` type used below that is a `dict` {"link_name": `str`, "link_url": `str`}.
+
+## `X`
+
+Corpora does not impose any additional constraints on the `X` data matrix. So it may be sparse or dense and any
+numeric `numpy.dtype`.
+
+## `uns`
+
+Recall that `uns` is a mapping with `str`s as keys. Corpora requires the following keys and values in `uns`:
+
+**Key**|**Value Type**|**Notes**
+-----|-----|-----
+title|`str`|Title of the dataset
```

Comment:
Given that this spec specifies the _encoding_ of data compliant with the corpora schema (defined elsewhere), I don't think it should include notes or additional semantic information like "title of the dataset".  Those should be completely defined in the Corpora schema spec.   I would constrain this spec to only that information related to encoding in H5AD, and push the semantic and more general schema info into the parent spec..

Reply:
Okay that seems reasonable.